engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -2091,8 +2091,11 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
           if (!commandLineInfo.localMode) {
 
             // see whether there is an app with the same name and user name running
-            String appNameAttributeName = StreamingApplication.DT_PREFIX + Context.DAGContext.APPLICATION_NAME.getName();
-            String appName = config.get(appNameAttributeName, appFactory.getName());
+            String appNameAttributeName = StreamingApplication.APEX_PREFIX + Context.DAGContext.APPLICATION_NAME.getName();
+            String appName = config.get(appNameAttributeName);
```

Comment:
When do you expect the old prefix to be gone? Should not it be supported till the next major release? addDeprecation can be removed at the same time when StreamingApplication.DT_PREFIX is removed.